src/pybind/mgr/dashboard_v2/tests/test_nodb.py

```diff
@@ -1,7 +1,5 @@
 from unittest import TestCase
-from unittest import skip
```

Comment:
Hey Sebastian,

I'm pretty sure there are reasons why this file belongs to the pull request but I don't see them currently. Could you please describe it shortly?

Reply:
I thought, no one would complain if I remove unused imports.

Reply:
Oh, don't get me wrong. I didn't want to complain (already approved your pull request) , it was just for my understanding. 

Okay, so you removed them, because they were unnecessary, great :)